persistence/src/main/resources/db/migration/schema/V1_2__CandidatesTable.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE candidates(
   id BIGINT AUTO_INCREMENT PRIMARY KEY,
   name VARCHAR(255),
-  email VARCHAR(255) ,
-  phone VARCHAR(12),
+  email VARCHAR(255) UNIQUE,
```

Comment:
Revert the changes in this file and create a new SQL migration file.

Do not forget, that the application can be deployed into a production / testing environment, where the database is not reset at every deploy.